streams/src/main/java/org/apache/kafka/streams/processor/internals/SinkNode.java

```diff
@@ -97,6 +96,10 @@ public void process(final K key, final V value) {
                                     keyClass,
                                     valueClass),
                     e);
+        } catch (final StreamsException e) {
```

Comment:
It looks like we've lost the context of the other exception, though. It doesn't include the message or stack from the previous exception.